app/services/proofing/aamva/proofer.rb

```diff
@@ -47,38 +47,46 @@ class Proofer
       # Instance methods
       def initialize(config)
         @config = Config.new(config)
+        @client = Aamva::VerificationClient.new(config)
       end
 
+      # @param applicant [Hash]
       def proof(applicant)
         aamva_applicant = Aamva::Applicant.from_proofer_applicant(applicant)
+        request = build_verification_request(aamva_applicant)
+        response = request&.send
 
-        response = Aamva::VerificationClient.new(
-          config,
-        ).send_verification_request(
-          applicant: aamva_applicant,
-        )
-
-        build_result_from_response(response, applicant[:state_id_jurisdiction])
+        build_result(request, response, applicant[:state_id_jurisdiction])
       rescue => exception
         Proofing::StateIdResult.new(
-          success: false, errors: {}, exception: exception, vendor_name: 'aamva:state_id',
-          transaction_id: nil, verified_attributes: [],
+          success: false,
+          errors: {},
+          exception: exception,
+          vendor_name: 'aamva:state_id',
+          transaction_id: nil,
+          verified_attributes: [],
+          requested_attributes: requested_attributes(request),
           jurisdiction_in_maintenance_window: jurisdiction_in_maintenance_window?(
             applicant[:state_id_jurisdiction],
-          )
+          ),
         )
       end
 
       private
 
-      def build_result_from_response(verification_response, jurisdiction)
+      def build_verification_request(applicant)
+        Aamva::VerificationClient.new(config)
+          .build_verification_request(applicant:)
+      end
+
+      def build_result(verification_request, verification_response, jurisdiction)
```

Comment:
nit: I think we've tended toward keyword args, esp. in cases where there is > 1 arg.

```suggestion
      def build_result(verification_request:, verification_response:, jurisdiction:)
```